internal/commands/app/create_inputs_test.go

```diff
@@ -127,7 +129,7 @@ func TestAppCreateInputsResolveName(t *testing.T) {
 			return nil, errors.New("realm client error")
 		}
 		inputs := createInputs{}
-		err := inputs.resolveName(nil, rc, appRemote{testApp.GroupID, testApp.ID})
+		err := inputs.resolveName(nil, rc, realm.AppFilter{GroupID: testApp.GroupID, App: testApp.ID})
```

Comment:
I think its best if we keep `AppFilter` to be using either `Name` or `ClientAppID` (as you saw, `FindApps` looks for matches on those rather than the actual ID values).  i know this is a test and is mocked, so things are likely working fine here, but best to try and keep it used in a way that helps document things